ci/do_ci.sh

```diff
@@ -33,12 +33,8 @@ fi
 
 export ENVOY_SRCDIR=$SOURCE_DIR
 
-# google cloud build times out when using full throttle.
-# additionally, we see builds killed due to OOM at high concurrency.
-export NUM_CPUS=10
```

Comment:
How stable is CI if we don't restrict the number of processes? Should we expect to see memory issues when we end up doing the 1.30 bump?

Reply:
I do not expect to see memory issues again since the runner we are using has such a vastly greater ratio of memory:cores. It's not so much about the number of processes, it's more an issue of how much memory is available for each process.

Reply:
makes sense, thanks!